TheAurorian/src/main/resources/assets/theaurorian/patchouli_books/the_aurorian_guide/zh_cn/entries/magicalitems/aurorianitesword.json

```
@@ -0,0 +1,13 @@
+{
+    "name": "极光剑",
+    "icon": "theaurorian:aurorianitesword",
+    "category": "3magicalitems",
+    "readbydefault": true,
+    "pages": [
+        {
+            "type": "spotlight",
+            "item": "theaurorian:aurorianitesword",
+            "text": "$(#008cff)极光剑$()在攻击时会让周围的所有人一同失重！默认情况下会有 30 秒的冷却（可在配置文件中调整）。$(br2)如果有盾牌或其它物品在你的副手上时你需要潜行以激活它。"
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
            "text": "$(#008cff)极光剑$()在攻击时会让周围的所有人一同失重！默认情况下会有 30 秒的冷却（可在配置文件中调整）。"
```
删掉的这句话需要实测。

---

没有问题，注意加个逗号，让句子变得通顺一点，还有不知道为什么如果....和....时 以外的让我感到古怪。

---

不是你古怪，这是我的问题

---

不是你古怪，是我的问题，改。
这句话我以前回复过一遍，后来被吞了